ntbs-service/Pages/Notifications/Delete.cshtml.cs

```diff
@@ -0,0 +1,65 @@
+using Microsoft.AspNetCore.Mvc;
+using ntbs_service.Models;
+using ntbs_service.Models.Enums;
+using ntbs_service.Models.Validations;
+using ntbs_service.Pages_Notifications;
+using ntbs_service.Services;
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+
+namespace ntbs_service.Pages.Notifications
+{
+    public class DeleteModel : NotificationModelBase
+    {
+        public ValidationService  ValidationService { get; set; }
+        [BindProperty]
+        [MaxLength(150)]
+        [RegularExpression(
+            ValidationRegexes.CharacterValidationWithNumbersForwardSlashAndNewLine, 
+            ErrorMessage = ValidationMessages.StringWithNumbersAndForwardSlashFormat)]
+        public string DeletionReason { get; set; }
+
+        public DeleteModel(INotificationService service, IAuthorizationService authorizationService) : base(service, authorizationService)
+        {
+            ValidationService = new ValidationService(this);

```

Comment:
Do we just want to be doing this assignment in the `NotificationModelBase` constructor??

Reply:
Yep think that's a good idea